indra/newview/llviewerwindowlistener.cpp

```diff
@@ -94,14 +88,36 @@ void LLViewerWindowListener::saveSnapshot(const LLSD& event) const
         TypeMap::const_iterator found = types.find(event["type"]);
         if (found == types.end())
         {
-            LL_ERRS("LLViewerWindowListener") << "LLViewerWindowListener::saveSnapshot(): "
-                                              << "unrecognized type " << event["type"] << LL_ENDL;
-        return;
+            return response.error(stringize("Unrecognized type ", std::quoted(event["type"].asString()), " [\"COLOR\"] or [\"DEPTH\"] is expected."));
```

Comment:
Oh good, thank you. The original use case for `LLEventAPI` was VITA automated testing, and for that it was acceptable to crash the viewer with an error when an entry point was engaged wrong. I've been trying to change those as I encounter them. I'm happy to see you doing the same.